### ISSUE TITLE
Render BBC news header component

### ISSUE THREAD

**Issue by jamesbhobbs** [bbc/simorgh#56]
Blocked by: https://github.com/bbc/simorgh/issues/54

Create a new component in simorgh to render a very basic BBC News header.

This should contain:
*A red background
*White text saying BBC NEWS
*A link from the text to https://www.bbc.com/news

Notes
*We will build a component library to move this component to in a separate issue in the near future, but first we will do a basic render of our articles
